dashboard/config/macroLoader.js

```diff
@@ -6,9 +6,18 @@ export default function(content) {
     emitFile(outputPath, fileContent) {
       context.emitFile(outputPath, fileContent);
 
-      return `module.exports = __webpack_public_path__ + ${JSON.stringify(
-        outputPath,
-      )}`;
+      // return `module.exports = __webpack_public_path__ + ${JSON.stringify(
+      //   outputPath,
+      // )};
+
+      // ¯\(°_o)/¯
+      // ¯\(°_o)/¯
+      // ¯\(°_o)/¯
+      // ¯\(°_o)/¯
+      // ¯\(°_o)/¯
+      // ¯\(°_o)/¯
+      // ¯\(°_o)/¯
```

Comment:
One of my life lessons is "Be funny out loud, don't be funny in code". Largely because once its in SCM, its suuper hard to take back.

Reply:
Haha, this is definitely very much WIP and not something I intend to find it's way into master. :)